deepchem/models/cnn.py

```diff
@@ -1,259 +0,0 @@
-import deepchem as dc
-import tensorflow as tf
-import numpy as np
-from deepchem.models import KerasModel
-from deepchem.models.layers import SwitchedDropout
-from deepchem.metrics import to_one_hot
-from tensorflow.keras.layers import Input, Dense, Reshape, Softmax, Dropout, Activation, Lambda
-import tensorflow.keras.layers as layers
-try:
-  from collections.abc import Sequence as SequenceCollection
-except:
-  from collections import Sequence as SequenceCollection
-
-
-class CNN(KerasModel):
-  """A 1, 2, or 3 dimensional convolutional network for either regression or classification.
```

Comment:
So are we removing TF CNN capability? Should we maybe create a keras_models module for the TF CNN?

Reply:
yes, as suggested by @peastman 

> I still think it would be best to remove the old TensorFlow CNN model. At the moment there are two different CNN models, each with a different test suite. There's no need for that. And part of the proof of the new model is that the old test suite continues to pass with the new implementation.